### ISSUE TITLE
Application freezes after a couple of seconds

### ISSUE THREAD

**Issue by L8zin** [flathub/com.valvesoftware.Steam#1056]
Version: 1.0.0.75

Operating System: Fedora Linux 37
KDE Plasma Version: 5.27.2
KDE Frameworks Version: 5.103.0
Qt Version: 5.15.8
Kernel Version: 6.1.14-200.fc37.x86_64 (64-bit)
Graphics Platform: Wayland
Processors: 24 × AMD Ryzen 9 3900X 12-Core Processor
Memory: 15.5 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3070/PCIe/SSE2
Manufacturer: ASUS

After being open for a few seconds, the steam client and all associated windows just freeze and become unresponsive. This has occured between reboots and started after updating today. When running the program through a terminal it doesnt output anything in particular. It does repeat this a few times before becoming unresponsive though:

`(steam:149): LIBDBUSMENU-GLIB-WARNING **: 19:58:24.783: Trying to remove a child that doesn't believe we're it's parent.`

System monitor also reports steam taking up nearly 800mb of RAM, but this might be normal behaviour.

**Comment by L8zin** [comment]
Yes that seems to be the exact issue i was having. Removing the animations fixed it

**Comment by JamesMcMahon** [comment]
> Yes that seems to be the exact issue i was having. Removing the animations fixed it

Can you elaborate on the fix? Not sure what "removing animations" means.
